nautobot/circuits/models.py

```diff
@@ -196,7 +196,7 @@ class Circuit(PrimaryModel, StatusModel):
 
     cid = models.CharField(max_length=100, verbose_name="Circuit ID")
     provider = models.ForeignKey(to="circuits.Provider", on_delete=models.PROTECT, related_name="circuits")
-    type = models.ForeignKey(to="CircuitType", on_delete=models.PROTECT, related_name="circuits")
+    circuit_type = models.ForeignKey(to="CircuitType", on_delete=models.PROTECT, related_name="circuits")
```

Comment:
This change will impact the CircuitFactory that I'm adding in #3115, just making a note here.